### PR TITLE
Project init complete 

### LINKED ISSUE #7: project initiailze
backstage import

### PR DESCRIPTION
**Pull request by YOOMYE89**
Description
close #7 

Checklist
 I have explained the need for this PR and the problem it solves
 I have explained the changes or the new features added to this PR
 I have added tests corresponding to this change
 I have updated the documentation to reflect this change
 I have verified that this change is backward compatible

### CODE REVIEW

**Comment by wijihoon** [comment]
```
backstage/cli 적용을 위해 테스트를 해보았지만, 
아래와 같은 이유로 동일한 오류가 지속 발생하였습니다.
현재와 같이 구조를 유지하는게 맞을거 같습니다.
```

## 구조
 1. Storybook 또한 package 안으로 넣어서 작업하려고 했으나, 종속성 문제로 하지 않는게 좋을 듯함.

 참고) 
Why is this not part of @backstage/core-components?
This separate storybook package exists because of dependency conflicts with @backstage/cli. It uses nohoist to avoid the conflicts, and since you can only use that in private packages it has to be separated out of @backstage/core-components.

https://github.com/backstage/backstage/tree/master/storybook

**Comment by hellices** [comment]
https://github.com/backstage/backstage/tree/master/storybook#storybook
pass one or more package```yarn storybook plugins/search```  & nohoist 로 용량을 줄일 수 있지 않을까? 검토

**Comment by YOOMYE89** [comment]
rebase 완료 및 모노레포 구성 완료

**Review by dev-shin — APPROVED** [review]

**Comment by hellices** [comment]
README.md update

```javascript
#모노레포 구성완료

yarn install

cd ./storybook && yarn install
cd .. # add!
yarn dev


```

**Review by wijihoon — APPROVED** [review]

File: plugins/home-react/README.md
[the remark this answers is not in the document]
Comment (hellices) [reply plugins/home-react/README.md:1]: 경로 오타 - plugin임

샘플이고 스토리북에서 보이는걸 확인하기 위한 용도.
향후 불필요하면 제거 예정

**Review by hellices — APPROVED** [review]

**Pull request merged**